index.bs

```diff
@@ -2234,26 +2235,34 @@ The [=remote end steps=] with |command parameters| are:
   1. Let |type| be the value of the <code>type</code> field of
      |command parameters|.
 
+  1. Let |reference context id| be the value of the <code>context</code> field
+     of |command parameters|, if present, or null otherwise.
+
+  1. If |reference context id| is not null, let |reference context| be the
```

Comment:
Question: Should we enforce the `reference context` to be a top-level browsing context? Right now it would accept any browsing context.